### PR TITLE
KH-539: Add default Superset datasets

### PR DESCRIPTION
**Pull request by enyachoke**
As a follow-up to initial work on https://mekomsolutions.atlassian.net/browse/KH-539, we needed to bring in some default datasets and database connection.

### CODE REVIEW

File: distro/configs/superset/dashboard-assets/datasets/PostgreSQL/concepts.yaml [hunk 1]
```diff
@@ -0,0 +1,137 @@
+table_name: concepts
+main_dttm_col: null
+description: null
+default_endpoint: null
+offset: 0
+cache_timeout: null
+schema: public
+sql: 'SELECT * FROM concepts '
```
Comment (Ruhanga) [inline distro/configs/superset/dashboard-assets/datasets/PostgreSQL/concepts.yaml:8]: Thanks for the PR @enyachoke, do we however need to consume these datasets, or these somewhat place holders/example dataset configs?

Comment (enyachoke) [reply distro/configs/superset/dashboard-assets/datasets/PostgreSQL/concepts.yaml:8]: We don't have to, but if we have any datasets which could conflict, we need to make sure we sync them up.

File: distro/configs/superset/dashboard-assets/databases/PostgreSQL.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+database_name: PostgreSQL
```
Comment (rbuisson) [inline distro/configs/superset/dashboard-assets/databases/PostgreSQL.yaml:1]: @enyachoke  this database should probably be named "Analytics", isn't it?
and should we update the file name to reflect that as well?

**Review by rbuisson — CHANGES_REQUESTED** [review]
Some changes requested.

Additionally, I have push on upstream the asset/ folder and create a Maven sub-module. Could you update your PR with the changes?

File: distro/configs/superset/dashboard-assets/datasets/PostgreSQL/concepts.yaml (hunk 1 above)
Comment (rbuisson) [reply distro/configs/superset/dashboard-assets/datasets/PostgreSQL/concepts.yaml]: Remove the trailing space in the `SELECT * FROM concepts `
@Ruhanga , yes, those are the default ones that will come with Ozone anyway.

**Review by rbuisson — APPROVED** [review]

**Pull request merged**